index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link href='https://fonts.googleapis.com/css?family=Roboto' rel='stylesheet'>
+    <title>Portfolio</title>
+</head>
+<body>
+    <div class="body-wrapper">
+        <nav class="header-flex">
+            <a class="logo" href=""><h5>Omar Mohammad</h5></a><span>
+            <img class="union" src="./assets/Union.png" alt='union-sign'/></span>
+        </nav>
+        <div class="container-flex">
+            <div class="intro-flex">
+                <h1>Omar Mohammad</h1>  
+                <h2>Software Developer</h2>        
+            </div>
+        <br>
+            <div class="about-flex">
+                <p class="about"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            </div>
+            <br/>
+            <hr>
+            <br/>
+            <div class="skills-flex">
+                <div class="languages">
+                    <h4>Languages</h4>
+                <ul >
+                    <li>JavaScript</li>
+                    <li>Ruby</li>
+                    <li>Css</li>
+                    <li>Html</li>
+                </ul>
+                </div>
+                <div class="frameworks">
+                    <h4>Frameworks</h4>
+                <ul>
+                    <li>Bootstrap</li>
+                    <li>Ruby on Rails</li>
+                    <li>RSPec</li>
+                    <li>CapyBara</li>
+                    <li>Selenium</li>
+                </ul>
+                </div>
+                <div class="skills">
+                    <h4>Skills</h4>
+                <ul>
+                    <li>Git</li>
+                    <li>Codekit</li>
+                    <li>Codepen</li>
+                    <li>Terminal</li>
+                    <li>GitHub</li>
+                </ul>
+                </div>
+            </div>
+            <hr>
+        </div>
+        
+
+                <div class="media-flex">
+                    <a href="#"><img alt= 'git' src="./assets/Github.svg"></a>
+                    <a href="#"><img alt= 'Vic' src="./assets/Victory.svg"></a>
+                    <a href="#"><img alt= 'Twitter' src="./assets/Twitter.svg"></a>
+                    <a href="#"><img alt= 'M' src="./assets/M.svg"></a>
+                </div>
+            
```

Comment:
- [ ] According to the requirements, You must stick to the design as much as possible (e.g. font, colors, images, tests, margins). So Kindly, Change the `<h4>` font color, Add some horizontal spaces between the elements to look similar to the [Figma Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A10)

| Your Design  | Figma Design  |   
| --- | --- |
| <img src="https://user-images.githubusercontent.com/45231276/177021115-7eff39b2-b004-416c-8663-ebd7ad35ec33.png" width="500" />  | <img src="https://user-images.githubusercontent.com/45231276/177021157-6dd75031-5d28-45ac-9b9d-f986cf26b2d0.png" /> |

Reply:
Sorry,..my bad.